prov/shm/src/smr_ep.c

```diff
@@ -209,6 +209,23 @@ static int smr_rx_src_comp_signal(struct smr_ep *ep, void *context, uint64_t fla
 		return ret;
 	ep->util_ep.rx_cq->wait->signal(ep->util_ep.rx_cq->wait);
 	return 0;
+
+}
+static int smr_verify_peer(struct smr_ep *ep, int peer_id)
+{
+	int ret = 0;
+
+	if (ep->region->map->peers[peer_id].peer.addr == FI_ADDR_UNSPEC) {
```

Comment:
nit: I would check for != ADDR_UNSPEC and return in that case.  This will allow the rest of the code to outdent